src/main/java/de/tum/in/www1/artemis/service/compass/umlmodel/UMLAssociation.java

```diff
@@ -2,6 +2,7 @@
 
 import de.tum.in.www1.artemis.service.compass.utils.CompassConfiguration;
 
+//TODO rename in Relation to stay consistent
```

Comment:
I changed it to UMLAssociation because this is the official term :-) We should rather rename all occurrences from Relation to Association

Reply:
done

Reply:
removed the TODO ^^